lib/api/bucketPut.js

```diff
@@ -59,7 +59,8 @@ function checkLocationConstraint(request, locationConstraint, log) {
         return { error: errorInstances.InvalidLocationConstraint.
             customizeDescription(errMsg) };
     }
-    if (locationConstraints[locationConstraintChecked].isCold) {
+    if (locationConstraints[locationConstraintChecked].isCold ||
+        locationConstraints[locationConstraintChecked].type === 'crr') {
```

Comment:
instead of doing the check based on the `type`, maybe we could to introduce a `feature` parameter, like isCold → `isCrr` ?